src/kf_scripts/prepare_sf.py

```diff
@@ -0,0 +1,160 @@
+import xarray as xr
+import os
+import numpy as np
+import yaml
+
+
+def sum_total_emissions(emissions, areas, mask):
```

Comment:
I think this should be imported instead of copied

Reply:
Agreed!